generator/test/error_test_data/embedded_object_primary_key.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:realm_common/realm_common.dart';
+
+//part 'embedded_object_primary_key.g.dart';
+
+@RealmModel(ObjectType.embedded)
+class _Bad {
+  @PrimaryKey()
```

Comment:
We can also test the generator for all the annotations `MapTo`, `@Indexed`, `@Ignored`.